ingestion/src/metadata/ingestion/source/bigquery.py

```diff
@@ -177,6 +182,16 @@ def fetch_sample_data(self, schema: str, table: str) -> Optional[TableData]:
 
         super().fetch_sample_data(schema, table)
 
+    def _get_database(self, database: str) -> Database:
+        if not database:
```

Comment:
Update the signature with `database: Optional[str]` if we expect that it can come empty